permissions/custom-checkbox-parameter.yml

```
@@ -0,0 +1,7 @@
+---
+name: "custom-checkbox-parameter"
+github: "jenkinsci/custom-checkbox-parameter-plugin"
+paths:
+- "org/jenkins-ci/plugins/custom-checkbox-parameter"
```

```suggestion
- "io/jenkins/plugins/custom-checkbox-parameter"
```

https://github.com/jenkinsci/custom-checkbox-parameter-plugin/blob/master/pom.xml#L10